src/App/Pages/Accounts/RegisterPageViewModel.cs

```diff
@@ -56,6 +56,13 @@ public bool AcceptPolicies
             set => SetProperty(ref _acceptPolicies, value);
         }
         
+        public Thickness GetSwitchMargin
```

Comment:
This is a property so the name should just be `SwitchMargin`

Reply:
Updated.